tests/kafkatest/services/kafka/kafka.py

```diff
@@ -193,10 +244,10 @@ def execute_reassign_partitions(self, reassignment):
 
         # create command
         cmd = "echo %s > %s && " % (json_str, json_file)
-        cmd += "/opt/kafka/bin/kafka-reassign-partitions.sh "\
-                "--zookeeper %(zk_connect)s "\
-                "--reassignment-json-file %(reassignment_file)s "\
-                "--execute" % {'zk_connect': self.zk.connect_setting(),
+        cmd += "/opt/%s/bin/kafka-reassign-partitions.sh " % kafka_dir(node)
+        cmd += "--zookeeper %(zk_connect)s "
+        cmd += "--reassignment-json-file %(reassignment_file)s "
+        cmd += "--execute" % {'zk_connect': self.zk.connect_setting(),
```

Comment:
Same issue here with string substitution only on the last piece of the string that was added.